bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "PublicLab Maps",
+  "homepage": "https://github.com/mridulnagpal/map_module",
+  "authors": [
+    "Public Lab"
+  ],
+  "description": "Test for PublicLab maps",
+  "main": "dist/PublicLab.Maps.js",
```

Comment:
Maybe this should be called `dist/Leaflet.BlurredLocation.js` - since we're hoping for a generalizable library that people find useful beyond PL?